libs/shared/create-course-form/src/lib/shared-create-course-form.scss

```diff
@@ -155,76 +162,8 @@ $error: #3f51b5;
       transform: translateX(-50%);
     }
   }
-}
-
-.course-form {
-  padding: 15px;
-  min-width: 700px;
-
-  .submit-form {
-    margin-top: 0.5rem !important;
-    padding: 0.5rem !important;
-    width: 48%;
-  }
-
-  .cancel-form {
-    padding: 0.5rem !important;
-    margin-top: 0.5rem !important;
-    width: 48%;
-  }
 
-  .dialog-list {
-    margin: 20px !important;
-    display: flex;
-    flex-direction: column;
-    justify-content: center;
-    font-family: 'Montserrat', sans-serif;
-  }
-
-  .chip-container {
-    display: flex;
-    flex-direction: row;
-    justify-content: center;
-    flex-wrap: wrap;
-    list-style: none;
-    .chip {
-      width: auto;
-      margin: 5px;
-    }
-  }
-
-  .select-button {
-    display: flex;
-    flex-direction: row;
-    justify-content: space-between;
-    // margin-bottom: 8.5px;
-  }
-
-  .error {
-    align-items: center;
-    color: $error;
-    display: flex;
-    font-style: inherit;
-    justify-content: center;
-    width: 100%;
-
-    &-icon {
-      color: $error;
-    }
-  }
-
-  .text-row {
-    display: flex;
-    justify-content: space-between;
-    .prerequisites,
-    .learning {
-      width: 100%;
-    }
-  }
-
-  .text-column {
-    width: 48%;
-    display: flex;
-    flex-direction: column;
+  .MuiButton-fullWidth {
+    padding: 0.8rem !important;
```

Comment:
I would prefer to avoid !important if possible. Write full class/elements path instead.